outreachy_proposal.md

```diff
@@ -0,0 +1,132 @@
+<hgroup>
+
+# Outreachy Proposal:
+
+## Crowdsourcing Translation for CHAOSS Diversity & Inclusion Badging
+
+</hgroup>
```

Comment:
```suggestion
# Outreachy Proposal: Crowdsourcing Translation for CHAOSS Diversity & Inclusion Badging
```

- My bad, let's stick with traditional `markdown` for now, we can explore all those tricks doen the road!